experiment_pages/experiment/map_rfid.py

```diff
@@ -1,544 +1,575 @@
+
+        simulate_rfid_button = CTkButton(self, text="Simulate RFID", compound=TOP,
+                                      width=15, command=self.add_random_rfid)
+        simulate_rfid_button.place(relx=0.80, rely=0.17, anchor=CENTER)
+
+        # Simulate All RFID Button
+        simulate_all_rfid_button = CTkButton(self, text="Simulate ALL RFID", compound=TOP,
+                                      width=15, command=self.simulate_all_rfid)
+        simulate_all_rfid_button.place(relx=0.80, rely=0.17, anchor=CENTER)
+
+        self.table_frame = CTkFrame(self)
+        self.table_frame.place(relx=0.15, rely=0.40, relheight=0.50, relwidth=0.80)
+        self.table_frame.grid_columnconfigure(0, weight= 1)
+        self.table_frame.grid_rowconfigure(0, weight= 1)
+
+        heading_style = Style()
+        heading_style.configure("Treeview.Heading", font=('Arial', 10))
+
+        columns = ('animal_id', 'rfid')
+        self.table = Treeview(
+            self.table_frame, columns=columns, show='headings', height=10, style='column.Treeview')
+
+        self.table.heading('animal_id', text='Animal ID')
+        self.table.heading('rfid', text='RFID')
+
+        self.table.grid(row=0, column=0, sticky='nsew')
+        self.table.grid_columnconfigure(0, weight = 1)
+        self.table.grid_rowconfigure(0, weight = 1)
+
+
+        scrollbar = CTkScrollbar(self.table_frame, orientation=VERTICAL, command=self.table.yview)
+        self.table.configure(yscroll=scrollbar.set)
+        scrollbar.grid(row=0, column=1, sticky='ns')
+
+        self.table.bind('<<TreeviewSelect>>', self.item_selected)
+
+        self.right_click = Menu(self, tearoff=0)
+        self.right_click.add_command(
+            label="Remove Selection(s)", command=self.remove_selected_items)
+        self.table.bind("<Button-3>", self.right_click_menu)
+
+        self.changer = ChangeRFIDDialog(parent, self)
+        self.serial_port_panel = SerialPortSelection(parent,self.serial_port_controller, self)
+
+        self.serial_port_button = CTkButton(self, text="Select Serial Port", compound=TOP,
+                                      width=15, command=self.open_serial_port_selection)
+        self.serial_port_button.place(relx=0.10, rely=0.95, anchor=CENTER)
+
+        self.change_rfid_button = CTkButton(self, text="Change RFID", compound=TOP,
+                                         width=15, command=self.open_change_rfid)
+        self.change_rfid_button.place(relx=0.40, rely=0.95, anchor=CENTER)
+
+        self.delete_button = CTkButton(self, text="Remove Selection(s)", compound=TOP,
+                                       width=20, command=self.remove_selected_items,
+                                       state="normal")  # Initialize button as disabled
+        self.delete_button.place(relx=0.70, rely=0.95, anchor=CENTER)
+
+        # Add Sacrifice button with normal state
+        self.sacrifice_button = CTkButton(self, text="Sacrifice Selected", compound=TOP,

```

Comment:
The sacrifice button does work, I am a fan that it works as a "selected" sacrifice option.